src/knoop/Node.hpp

```diff
@@ -25,6 +25,10 @@ class Node {
   #include "knoop/Node/src/push_front.hpp"
   #include "knoop/Node/src/pop_back.hpp"
   #include "knoop/Node/src/pop_front.hpp"
+  #include "knoop/Node/src/front.hpp"
+  #include "knoop/Node/src/back.hpp"
+  #include "knoop/Node/src/begin.hpp"
```

Comment:
I have mixed feelings about the iterator methods. I can't see the header, but I assume these return iterators over the list.

A map defines iteration as well. It seems strange that the built in begin and end would prefer the list interface. I think limit access to iteration by way of the `list` method (or perhaps a map method in the future) is more explicit and less likely to surprise.

Reply:
You bring up a good point. (You should be able to see the headers now.) It probably is better to be explicit about what we are iterating over.